### PR TITLE
save non-string object on LogRecord

### PR DESCRIPTION
**Pull request by jasonaguilon-wf**
So that a handler can access the original object instead of just its
`toString()`.

This enables a couple different use cases:
- sending an arbitrary object to the logging methods on `window.console`
- handler specific serialization

### CODE REVIEW

File: lib/logging.dart [hunk 1]
```diff
@@ -334,6 +340,7 @@ class Level implements Comparable<Level> {
 class LogRecord {
   final Level level;
   final String message;
+  final Object object;
```
Comment (sigmundch) [inline lib/logging.dart]: would you mind adding a short comment here to document this new field as well?

**Comment by sigmundch** [comment]
lgtm - thanks for adding a tests together with your change!

We would just need to update the pubspec.yaml to bump the minor version (to 0.11.3) and add an entry on the CHANGELOG.md before we can publish it. Up to you, but feel free to do it with this change if you like.

**Comment by jasonaguilon-wf** [comment]
Feedback addressed in the new commits. Thank you @sigmundch for taking a look at this.

**Comment by sigmundch** [comment]
Thanks @jasonaguilon-wf !

**Comment by jasonaguilon-wf** [comment]
@sigmundch Do you have an ETA on when I could have a release of this cut and published?

Thanks in advance!

**Comment by sigmundch** [comment]
@jasonaguilon-wf thanks for the reminder! I forgot that very last step of publishing it :) - it should be live now.

**Comment by jasonaguilon-wf** [comment]
Perfect, thanks again @sigmundch !

**Pull request merged**